serverless.js

```diff
@@ -30,6 +22,11 @@ module.exports = {
     region: `\${env:AWS_REGION, "us-east-1"}`,
     stage: `\${env:SLS_STAGE, "development"}`,
   },
+  package: {
+    individually: true,
+    // we don't need this because of ncc
```

Comment:
I see, ncc only includes what is needed anyway. But do we need to explicitly exclude things here then?

Reply:
This configuration is for serverless. Since `ncc` is packaging the source code serverless doesn't need to remove `devDependencies`.